engine/src/main/java/org/entando/entando/aps/system/services/oauth2/OAuth2TokenDAO.java

```diff
@@ -190,6 +193,18 @@ public void storeAccessToken(OAuth2AccessToken accessToken, OAuth2Authentication
             stat.executeUpdate();
             conn.commit();
         } catch (Exception t) {
+            CharArrayWriter stackTraceWriter = new CharArrayWriter();
```

Comment:
I don't think we can merge this. Looks like a potential security problem. We'll have to find the underlying race condition.